dsm/resource_sobject.go

```diff
@@ -223,7 +248,9 @@ func createSO(ctx context.Context, d *schema.ResourceData, m interface{}) diag.D
 			"radix": d.Get("fpe_radix").(int),
 		}
 	}
-
+	if err := d.Get("custom_metadata").(map[string]interface{}); len(err) > 0 {
+		security_object["custom_metadata"] = err
```

Comment:
Why are we assigning err to a field of API call?

Reply:
In the if condition err will be assigned to custom_metadata, if it has length > 0 then it will assign directly to the security object instead of calling again.

In another way I can do like this:

security_object["custom_metadata"] = d.Get("custom_metadata").(map[string]interface{})

But I am already assigning it to err, hence did it.